tox.ini

```diff
@@ -59,7 +59,8 @@ deps:
 commands =
     pip install pip --upgrade
     pip install tox --upgrade
-    pyutils-jira --config-file-path jira.cfg --target-versions "vfuture,4.18.1,4.18.2,4.18.3,4.18.4,4.18.5,4.19.0" --verbose
+    pyutils-jira --config-file-path jira.cfg --target-versions "4.18.1,4.18.2,4.18.3,4.18.4,4.18.5" --verbose
```

Comment:
this one also should work, I think:
```
    pyutils-jira --config-file-path jira.cfg --target-versions "4.18." --verbose
```

Reply:
let's try :)

Reply:
sorry for the noise - it seems not supported yet :( so need to revert full line back `4.18.1,4.18.2,4.18.3,4.18.4,4.18.5`

Reply:
no worries, added them back, but dropped the 4.18.1 since it was released this week